.github/workflows/ci.yml

```diff
@@ -327,6 +220,73 @@ jobs:
         # pwsh ./test/Scripts.Tests/test-pack-contents.ps1 accept
         run: ./test/Scripts.Tests/test-pack-contents.ps1
 
+  windows-smoke-test:
+    needs: [build]
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # 2022.1.0a12 removed until S.T.J issues fixed
+        unity-version: [2019.4.34f1, 2020.3.27f1, 2021.2.10f1]
+        include:
+          - os: windows-latest
+            unity-config-path: C:/ProgramData/Unity/config/
+          - unity-version: 2019.4.34f1
+            unity-version-changeset: 6a9faed444f2
+          - unity-version: 2020.3.27f1
+            unity-version-changeset: e759542391ea
+          - unity-version: 2021.2.10f1
+            unity-version-changeset: ee872746220e
+    defaults:
+      run:
+        shell: pwsh
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+
+      - name: Download UPM package
+        uses: actions/download-artifact@v2
+        with:
+          # Artifact name is the commit sha. Which is what craft uses to find the relevant artifact.
+          name: ${{ github.sha }}
+
+      - name: Setup Unity
+        uses: getsentry/setup-unity@46c2e082d98cc3a825a5b59038cb31705fe9ff56
+        with:
+          unity-version: ${{ matrix.unity-version }}
+          unity-version-changeset: ${{ matrix.unity-version-changeset }}
+          unity-modules: windows-il2cpp
```

Comment:
Windows runners can't use containers because of licensing & the limitations of Windows Server 2022 (no support for WSL2), see https://github.com/actions/runner/issues/904

Therefore we stay with the standard unity setup here